src/lib/parser/templates.ts

```diff
@@ -0,0 +1,4 @@
+import { customObject } from '../../lib/parser/defaults/CustomObjectsMd';
```

Comment:
Wouldn't this make more sense if is: `from '../defaults/CustomObjectsMd'` ? No need to go so many levels back, you're already in the `parser` folder.